global.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Adding this was necessary to get the project to run on my machine that has both the dotnet 3.1 and 5 runtimes. @Keboo Do you think it should be included in the project or is there a better way to handle multiple runtimes?

Relevant documentation:
https://docs.microsoft.com/en-us/dotnet/core/versions/selection#the-sdk-uses-the-latest-installed-version

Reply:
It would be better to set it up to [roll forward](https://docs.microsoft.com/en-us/dotnet/core/tools/global-json?tabs=netcore3x#rollforward) rather than being pinned to an exact 3.1 release